[GSoD 2021] Added CONTRIBUTING.md

iamrajiv opened the pull request:
Related: wechaty/docusaurus#1012

![Screenshot 2021-07-06 at 9 48 28 PM](https://user-images.githubusercontent.com/42106787/124634412-f23c3680-dea3-11eb-97e6-a2acd4c67448.jpg)
![Screenshot 2021-07-06 at 9 48 37 PM](https://user-images.githubusercontent.com/42106787/124634397-ee101900-dea3-11eb-9c29-003ce1deb225.jpg)

iamrajiv committed:
added contributing.md

Signed-off-by: Rajiv Ranjan Singh <rajivperfect007@gmail.com>

iamrajiv commented:
@proudofsimin @Rohitesh-Kumar-Jain Please take a look.

proudofsimin commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,138 @@
+
+## How to [contribute](https://github.com/wechaty/openapi/blob/master/CONTRIBUTING.md)
+
+- Write a blog about Wechaty OpenAPI.
+- Create a tutorial for Wechaty OpenAPI.
```
How do you think if we add a link to [General Wechaty blog post guidelines](https://github.com/wechaty/wechaty.js.org)?

iamrajiv replied:
@proudofsimin Added the link.

proudofsimin reviewed (commented):
This looks so nice!

iamrajiv committed:
added General Wechaty blog post guidelines link

Signed-off-by: Rajiv Ranjan Singh <rajivperfect007@gmail.com>

lijiarui commented:
Thanks for improving this!

I have a suggestion to combine all of the contributing guides together to our docs, see: https://github.com/wechaty/wechaty.js.org/issues/1111

iamrajiv commented:
@lijiarui Great! so repo won't have CONTRIBUTING.md file?

huan reviewed (changes requested):
Please link to our official docs website at https://wechaty.js.org/docs/contributing/

iamrajiv committed:
Update CONTRIBUTING.md

iamrajiv commented:
@huan I have updated to contributing file of this PR and added the https://wechaty.js.org/docs/contributing/ link. I have made PR for improvement of contributing  guidelines here wechaty/wechaty.js.org#1276, please review

huan reviewed (approved):
LGTM

huan commented:
It's a great CONTRIBUTING file, thank you very much!
